projects/core_presentation.md

```diff
@@ -0,0 +1,51 @@
+Core Project Presentations
+==========================
+
+In order to increase the continuous learning of our team, and provide fodder for our marketing, we encourage Core Project Presentations
+
+At this point, there is no Template for such a presentation, but it should include the following:
+
+_NOTE: Total presentation should be between 50 minutes and 90 minutes.  Depending on the project, more time may be spent in one section than the other, but each should be considered._
+
+## Quick History and Context _(5-10 minutes)_
+
+* What was the motivation for initiating the project?
+* What is the hypothesis we’re trying to prove?
+* What was done before the current phase?
+  * If in the Craftsmanship Phase, what was done in the Planning & Exploration Phases
+  * If this is a continuation of a previous project, what was done in the previous project and what was done between projects
+* Who have the players been (and who are they currently)?
+
+## Customer Value Add _(10-20 minutes)_
+
+* What have we delivered to the customer?
```

Comment:
This part feels a little like a status report.  If it is in fact a summary report, then it might be worth adding a couple of template lines for steps/iterations that were discarded or took an unexpected direction for whatever reason.